### PR TITLE
Create a list

### LINKED ISSUE #1: Userstory #1 : Create a list
## Value proposition

---

**As a** user
**I need** to see a List with names of social facilities
**to be able to** find the name of the facility that I need. 

## Description

---

Image

## Acceptance criteria

---

- [ ] The user can see the names of all social facilities
- [ ] the list should have it's own headline
- [ ] the names are displayed one below the other

## Tasks

---

- [x] create remote Repository on Github
- [x] connect template and remote Repository
- [x] open feature branch
- [x] create dummy data in json file (with properties:  name, image, adress( with google-link),target, requirements, category)
- [x] create page with a header 
- [x] fetch dummy data (only the names)
- [x] create unordered list with one list item for each facility name

## Complexity Estimation

---

S

### PR DESCRIPTION
**Pull request by BucherLara**
Please check out my first feature! Thanks a lot!

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+
+  if (error) return <div>failed to load</div>;
+  if (!data) return <div>loading...</div>;
+
+  return (
+    <>
+      <Head>
+        <title>App-Name</title>
+      </Head>
+      <header></header>
```
Comment (Kheirah) [inline pages/index.js]: As a general rule of thumb, leave only that in the code which is immediately useful. An empty tag doesn't do anything for the app. Add it back in when it becomes useful :)

**Review by Kheirah — APPROVED** [review]
Good start! Keep it up!

File: pages/index.js [hunk 2]
```diff
@@ -1,3 +1,36 @@
+import fetcher from "../helpers/fetcher";
+import useSWR from "swr";
+import styled from "styled-components";
+import Head from "next/head";
+
 export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+  const { data, error } = useSWR("api/facilities", fetcher);
```
Comment (shebtastic) [inline pages/index.js]:
```suggestion
  const { data, error } = useSWR("/api/facilities", fetcher);
```

it's always a good idea to use absolute paths for endpoints

File: db.json [hunk 3]
```diff
@@ -0,0 +1,132 @@
+[
+  {
+    "id": 1,
+    "name": "Pik As",
+    "category": "Schlafen",
+    "adress": "Neustädter Str. 31 a, 20355 Hamburg, https://www.google.com/maps/place/Pik+As/@53.553246,9.980664,17z/data=!3m1!4b1!4m5!3m4!1s0x47b18f10d01e241f:0xabb10810d7b57cb4!8m2!3d53.5532528!4d9.9828532",
+    "target": "Für Männer",
+    "requirements": "Keine",
+    "image": "",
+    "value": "Sofort, Hund"
```
Comment (shebtastic) [inline db.json]:
```suggestion
    "tags": ["Sofort", "Hund"]
```

**Review by shebtastic — CHANGES_REQUESTED** [review]

**Pull request merged**